src/main/webapp/i18n/de/competency.json

```
@@ -14,14 +14,20 @@
             "optionalDescription": "Optionale Kompetenzen müssen nicht absolviert werden um den Lernpfad zu vervollständigen. Sie dienen entweder als zusätzliche Übung oder gehen über die erwarteten Inhalte des Kurses hinaus.",
             "course": "Kurs",
             "progress": "Fortschritt",
-            "progressDescription": "Der Prozentsatz an abgeschlossenen Vorlesungseinheiten und Übungen, die mit dieser Kompetenz verknüpft sind.",
-            "confidence": "Konfidenz",
-            "confidenceDescription": "Die durchschnittliche Bewertung in allen verknüpften Übungen im Verhältnis zu dem vom Lehrenden festgelegten Schwellenwert für die Beherrschung der Kompetenz.",
+            "progressDescription": "Der Fortschritt an Inhalten, die mit dieser Kompetenz verknüpft sind. Er basiert auf dem Prozentsatz an abgeschlossenen Vorlesungseinheiten und den erreichten Punkten in den Übungen.",
             "mastery": "Kompetenzbeherrschung",
-            "masteryDescription": "Ein gewichteter Leistungsindex aus Fortschritt und Konfidenz, der Deine Entwicklung auf dem Weg zur Beherrschung der Kompetenz veranschaulicht.",
+            "masteryDescription": "Ein gewichteter Leistungsindex aus deinem Fortschritt, der Deine Entwicklung auf dem Weg zur Beherrschung der Kompetenz veranschaulicht.",
             "masteredStudents": "Studierende mit Kompetenz",
             "mastered": "Abgeschlossen",
             "competencies": "Kompetenzen",
+            "confidenceReasons": {
+                "NO_REASON": "Die Kompetenzbeherrschung ist etwa der Fortschritt",
```

No strong opinion, does not need to be changed 

```suggestion
                "NO_REASON": "Die Kompetenzbeherrschung entspricht in etwa dem Fortschritt",
```